packages/bot-web-ui/src/components/flyout/flyout-block.jsx

```diff
@@ -8,7 +8,7 @@ const FlyoutBlock = ({ initBlockWorkspace, block_node, should_center_block, shou
 
     React.useEffect(() => {
         initBlockWorkspace(el_block_workspace, block_node, should_center_block);
-    }, []);
+    }, [should_center_block, block_node, initBlockWorkspace]);
```

Comment:
because of the nature of useEffects with empty dependency array, are you sure that the added dependencies will not change?

Reply:
We use empty array to have the componentDidMount effect in our functional components but the warning that we are receiving is: "react hook useeffect has missing dependencies: 'block_node', 'initBlockWorkspace' and 'should_center_block'. Either include it or remove the dependency array.".
The recommended solution in the warning was adding the dependencies which changes our componentDidMount to componentDidUpdate. Any recommendations from your side ?

Reply:
in some cases it's not a bad idea to use `// eslint-disable-next-line react-hooks/exhaustive-deps` to just ignore the warning